starlarktime/time.go

```diff
@@ -0,0 +1,507 @@
+package starlarktime
```

Comment:
What happened to the doc comment?

Reply:
@alandonovan it has been removed since it was not in a standard format and there is no other module with this kind of documentation